upload-api/buckets/car-store.js

```diff
@@ -81,3 +81,39 @@ export function useCarStore(s3, bucketName) {
     },
   }
 }
+
+/**
+ * compose many car stores.
+ * store#createUploadUrl is from first store.
+ * store#has will check stores in order until 0-1 `true` are found.
+ * 
+ * @param  {import('@web3-storage/upload-api').CarStoreBucket} carStore 
+ * @param  {Array<import('@web3-storage/upload-api').CarStoreBucket>} moreCarStores 
+ */
+export function composeCarStores(carStore, ...moreCarStores) {
```

Comment:
I would call it something else, where is more clear to it fallbacking only on `has`, being everything else the first. More a `fallbackHasComposeCarStore`? not sure :)

Reply:
renaming to `composeCarStoresWithOrderedHas`